src/utils.ts

```diff
@@ -520,4 +522,40 @@ const katexMarkedExtension = {
     }),
 };
 
+// Overwrites the rendering of heading tokens.
+// Since there are possible non ASCII characters in headings,
+// we escape them by replacing them with dashes and,
+// trim eventual dashes on each side of the string.
```

Comment:
Do you have a concrete example of where this happens?

Reply:
http://127.0.0.1:3000/seeds/clients.radicle.xyz/rad:git:hnrkmg77m8tfzj4gi4pa4mbhgysfgzwntjpao/tree/04aa53a1711ff9deaf9e9da5e896bc9b7d1b3995

Here are a few headings that take emojis, and then are a PITA